is-within-range/main.js

```diff
@@ -0,0 +1,15 @@
+//declare a function named isWithinRange with two parameters
+//first is a number and second is an object
+function isWithinRange(number, rangeObject) {
+  //declare a for loop that iterates through the object using a variable named key
+  for(var key in rangeObject) {
```

Comment:
It doesn't look as though this loop is doing anything for you, you can probably just remove it 👍

Reply:
youre right lol